pan3d/dataset_builder.py

```diff
@@ -257,6 +259,53 @@ def mesh(
     # Internal methods
     # -----------------------------------------------------
 
+    def _load_dataset(self, dataset_info):
+        ds = None
+        if dataset_info is not None:
+            source = dataset_info.get("source")
+            if source == "pangeo":
+                if self._pangeo:
```

Comment:
still don't like those blocks

maybe like 

```python
if not self._pangeo: # still not sure about those flat flags
   raise ValueError(
      "Pangeo module not enabled. Set pangeo=true to load this dataset."
   )
from pan3d.pangeo_forge import load_dataset
ds = load_dataset(dataset_info["id"])
```

Reply:
see my other try/except comment which might be even better... 

Still don't get those flags